### PR TITLE
feat(zig/fib): index stdin with no args

### PR DESCRIPTION
**Pull request by rvolosatovs**
Refs #23

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -159,7 +159,7 @@
 
       fibonacci-zig-wasm = final.stdenv.mkDerivation {
         pname = "fibonacci";
-        version = "0.2.0";
```
Comment (bstrie) [inline flake.nix:162]: Hm, do I need to update this in any other examples I touch too?

Comment (rvolosatovs) [reply flake.nix:162]: For rust, we parse Cargo.toml and use that version.
For languages without a similar mechanism (like Zig) - yes

Comment (bstrie) [reply flake.nix:162]: Do I need to be updating the version in Cargo.toml every time I make a change to any of the Rust examples, then? Is this the tag that gets used for uploading to Drawbridge?

**Review by bstrie — APPROVED** [review]

File: flake.nix (hunk 1 above)
Comment (rvolosatovs) [reply flake.nix:162]: Yes and Yes (will be)

**Pull request merged**